crates/wasmtime/src/runtime/fiber.rs

```diff
@@ -0,0 +1,700 @@
+#![deny(unsafe_op_in_unsafe_fn)]
+
+use {
```

Comment:
FWIW this style of imports is currently largely nonexistent within Wasmtime. Would you be ok reformatting this in a style more consistent with the rest of Wasmtime?